api/securitytest/run.go

```diff
@@ -103,7 +103,7 @@ func (results *RunAllInfo) runGenericScans(enryScan SecTestScanInfo) error {
 		go func(genericTest *types.SecurityTest) {
 			defer wg.Done()
 			newGenericScan := SecTestScanInfo{}
-			if err := newGenericScan.New(enryScan.RID, enryScan.URL, enryScan.Branch, genericTest.Name); err != nil {
+			if err := newGenericScan.New(enryScan.RID, enryScan.URL, enryScan.Branch, genericTest.Name, nil); err != nil {
```

Comment:
What is this `nil`? It sounds like a magical number example. What do you think of adding a variable describing what it is meant and setting nil to it? 

```go
languangesToExclude := nil
if err := newGenericScan.New(enryScan.RID, enryScan.URL, enryScan.Branch, genericTest.Name, languangesToExclude)
```

Reply:
Indeed it's a bit too generic, creating a new variable as nil wouldn't compile, so i just emptied the previous object by assigning to nil. Also added a comment in order to make it a bit more clear.

the language exclusions struct is only used on the enryScan (In order to validate if it should append the language to the repoLanguages or not). Due to the scan creation method being generic it is also used to create the following scans, but they don't won't use the languageExclusions var for anything.
 
```go
// LanguageExclusions is only utilized on first scan (enryScan) therefore set as nil
enryScan.LanguageExclusions = nil
```